### PR TITLE
PUT on a field with a false value should be persisted

### PR DESCRIPTION
**Pull request by adriana-saroz**

### CODE REVIEW

File: src/test/access/exposed.ts [hunk 1]
```diff
@@ -127,7 +128,7 @@ describe('exposed fields', () => {
   });
 });
 
-describe('fieldProjection', () => {
+describe('Field projection', () => {
```
Comment (gedaiu) [inline src/test/access/exposed.ts]: OCD :smile:

Comment (adriana-saroz) [inline src/test/access/exposed.ts]: I admit :whale:

File: src/test/access/exposed.ts [hunk 2]
```diff
@@ -235,6 +236,44 @@ describe('Exposing fields', () => {
     });
   });
 
+  it('should set exposed fields that were not part of the payload to null when trying to PUT', (done: MochaDone) => {
```
Comment (gedaiu) [inline src/test/access/exposed.ts]: maybe `it should set exposed missing payload fields to null on PUT` ?

File: src/test/access/exposed.ts [hunk 3]
```diff
@@ -235,6 +236,44 @@ describe('Exposing fields', () => {
...
+    item.save(() => {
+      let id = item._id.toString();
+
+      test.server.createRequest()
+        .payload('exposetest', { name: 'new name' })
+        .put('/exposetests/' + id)
+        .then((res: any) => {
+          test.model.findById(id, (err: any, record: any) => {
+            record.name.should.equal('new name');
+            should(record.flag).equal(null);
```
Comment (gedaiu) [inline src/test/access/exposed.ts]: should we check for name too?

File: src/test/access/exposed.ts [hunk 4]
```diff
@@ -235,6 +236,44 @@ describe('Exposing fields', () => {
...
+    let item = new test.model({ name: 'some name', value: 'some value', flag: true });
+
+    item.save(() => {
+      let id = item._id.toString();
+      test.server.createRequest()
+        .payload('exposetest', { flag: false })
+        .put('/exposetests/' + id)
+        .then((res: any) => {
+          test.model.findById(id, (err: any, record: any) => {
+            record.flag.should.equal(false);
```
Comment (gedaiu) [inline src/test/access/exposed.ts]: should we check for name and value too?

**Pull request merged**